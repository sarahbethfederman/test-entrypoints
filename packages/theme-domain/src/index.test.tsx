import * as React from 'react';
import { mount } from 'enzyme';
import ThemeDomain from './index';

let element;

describe('ThemeDomain', () => {
  beforeEach(() => {
    element = mount(<ThemeDomain />);
  });

  it('should mount', () => {
    expect(element.find(ThemeDomain)).toHaveLength(1);
    expect(element.find(ThemeDomain)).toMatchSnapshot();
  });
});
