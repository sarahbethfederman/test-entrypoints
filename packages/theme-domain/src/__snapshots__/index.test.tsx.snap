// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ThemeDomain should mount 1`] = `
.c0 {
  color: #474747;
}

<Theme>
  <ThemeProvider
    theme={
      Object {
        "colors": Object {
          "error": Object {
            "100": "#ffd1da",
            "200": "#ffa4b6",
            "25": "#fff3f5",
            "300": "#ff7692",
            "400": "#ff496e",
            "50": "#ffe8ec",
            "500": "#ff1c4a",
            "600": "#cc163b",
            "700": "#99102c",
            "800": "#660b1d",
            "900": "#33050e",
            "950": "#190207",
          },
          "info": Object {
            "100": "#dae7f7",
            "200": "#b5d0ef",
            "25": "#f5f9fd",
            "300": "#91b9e8",
            "400": "#6ca2e0",
            "50": "#ecf3fb",
            "500": "#488bd9",
            "600": "#396fad",
            "700": "#2b5382",
            "800": "#1c3756",
            "900": "#0e1b2b",
            "950": "#070d15",
          },
          "primary": Object {
            "100": "#ceedcc",
            "200": "#9edc99",
            "25": "#f2faf2",
            "300": "#6eca66",
            "400": "#3eb932",
            "50": "#e6f6e5",
            "500": "#0ea800",
            "600": "#0b8600",
            "700": "#086400",
            "800": "#075c00",
            "900": "#054300",
            "950": "#043200",
          },
          "secondary": Object {
            "100": "#ced2da",
            "200": "#9da5b5",
            "25": "#f2f3f5",
            "300": "#6c7990",
            "400": "#3b4c6b",
            "50": "#e6e8ec",
            "500": "#0b2047",
            "600": "#06193a",
            "700": "#05132b",
            "800": "#030c1d",
            "900": "#02060e",
            "950": "#054300",
          },
          "shade": Object {
            "0": "#ffffff",
            "100": "#e3e3e3",
            "1000": "#000000",
            "200": "#c8c8c8",
            "25": "#f8f8f8",
            "300": "#adadad",
            "400": "#929292",
            "50": "#f1f1f1",
            "500": "#777777",
            "600": "#5f5f5f",
            "700": "#474747",
            "800": "#2f2f2f",
            "900": "#171717",
            "950": "#0b0b0b",
          },
          "success": Object {
            "100": "#dbf2cc",
            "200": "#b8e599",
            "25": "#f6fbf2",
            "300": "#94d966",
            "400": "#71cc32",
            "50": "#edf8e5",
            "500": "#4ec000",
            "600": "#3e9900",
            "700": "#2e7300",
            "800": "#1f4c00",
            "900": "#0f2600",
            "950": "#071300",
          },
          "tertiary": Object {
            "100": "#d8dade",
            "200": "#b1b5bd",
            "25": "#f5f5f6",
            "300": "#8a909c",
            "400": "#626b7b",
            "50": "#ebecee",
            "500": "#3c475b",
            "600": "#2d374b",
            "700": "#222938",
            "800": "#191e2a",
            "900": "#0e1117",
            "950": "#050709",
          },
          "warn": Object {
            "100": "#fdedd3",
            "200": "#fbdba7",
            "25": "#fefaf4",
            "300": "#f9c97b",
            "400": "#f7b74e",
            "50": "#fef6e9",
            "500": "#f5a623",
            "600": "#c4841c",
            "700": "#936315",
            "800": "#62420e",
            "900": "#312107",
            "950": "#181003",
          },
        },
        "logo": Object {
          "logoName": "DomainLogo",
        },
        "typography": Object {
          "body": Object {
            "fontFamily": "Larsseit",
          },
          "heading": Object {
            "fontFamily": "Larsseit ExtraBold",
          },
        },
      }
    }
  >
    <styled.div>
      <div
        className="c0"
      />
    </styled.div>
  </ThemeProvider>
</Theme>
`;
